src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,167 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static RegistrationService registration;
+
+    @BeforeAll
+    static void beforeAll() {
+        registration = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_additionUsers_ok() {
+        User firstUser = new User();
+        firstUser.setLogin("Login");
+        firstUser.setPassword("passWord1");
+        firstUser.setAge(MIN_AGE);
+
+        User secondUser = new User();
+        secondUser.setLogin("login");
+        secondUser.setPassword("passWord1");
+        secondUser.setAge(MIN_AGE);
+
+        boolean expect = firstUser.equals(registration.register(firstUser));
+        assertTrue(expect);
+        expect = secondUser.equals(registration.register(secondUser));
+        assertTrue(expect);
+    }
```

Comment:
this test in redundant, but you can leave it
```suggestion
    @Test
    void register_additionUsers_ok() {
        User firstUser = new User();
        firstUser.setLogin("Login");
        firstUser.setPassword("passWord1");
        firstUser.setAge(MIN_AGE);

        User secondUser = new User();
        secondUser.setLogin("login");
        secondUser.setPassword("passWord1");
        secondUser.setAge(MIN_AGE);

        boolean expected = firstUser.equals(registration.register(firstUser));
        assertTrue(expect);
        expected = secondUser.equals(registration.register(secondUser));
        assertTrue(expect);
    }
```
change `expect` to `expected` in all tests